x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/rule_status.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useMemo } from 'react';
+import { EuiBadge } from '@elastic/eui';
+import { getOr } from 'lodash/fp';
+
+import styled from 'styled-components';
+import { DefaultDraggable } from '../../../../../common/components/draggables';
+
+const mapping = {
+  open: 'primary',
+  'in-progress': 'warning',
+  closed: 'default',
+};
+
+const StyledEuiBadge = styled(EuiBadge)`
+  text-transform: capitalize;
+`;
+
+interface Props {
+  contextId: string;
+  eventId: string;
+  fieldName: string;
+  value: string | number | undefined | null;
+}
+
+export const RenderRuleStatus: React.FC<Props> = ({ contextId, eventId, fieldName, value }) => {
```

Comment:
for the basic component, we use naming as `RuleStatusComponent` and we usually don't export them to make sure we are using memorized components